frontend/src/scenes/session-recordings/player/PlayerEvents.tsx

```diff
@@ -1,5 +1,72 @@
 import React from 'react'
+import { Col, Input, Row } from 'antd'
+import { SearchOutlined } from '@ant-design/icons'
+import { useActions, useValues } from 'kea'
+import VirtualizedList, { ListRowProps } from 'react-virtualized/dist/commonjs/List'
+import { AutoSizer } from 'react-virtualized/dist/commonjs/AutoSizer'
+import { CellMeasurer } from 'react-virtualized/dist/commonjs/CellMeasurer'
+import { eventsListLogic } from 'scenes/session-recordings/player/eventsListLogic'
+import { ActionIcon, AutocaptureIcon, EventIcon, PageleaveIcon, PageviewIcon } from 'lib/components/icons'
+import { eventToName } from 'lib/utils'
 
 export function PlayerEvents(): JSX.Element {
-    return <div>Player Events</div>
+    const { localFilters, listEvents, cellMeasurerCache } = useValues(eventsListLogic)
+    const { setLocalFilters } = useActions(eventsListLogic)
+
+    function Event({ index, style, key, parent }: ListRowProps): JSX.Element {
+        const event = listEvents[index]
+
+        const renderIcon = (): JSX.Element => {
+            if (event.event === '$pageview') {
+                return <PageviewIcon />
+            }
+            if (event.event === '$pageleave') {
+                return <PageleaveIcon />
+            }
+            if (event.event === '$autocapture') {
+                return <AutocaptureIcon />
+            }
+            if (event.event.startsWith('$')) {
+                return <EventIcon />
+            }
+            return <ActionIcon />
```

Comment:
I think this would give custom events the action icon. I think we were looking at using the 'code' icon for those.

Reply:
There doesn't seem to be a way to distinguish events, custom_events, and actions from the events results that are returned from the `api/events` endpoint. For now, I'll default all non pageevents or autocapture events to show the event icon, and make a note to cut a PR that attaches the `action_id` to all events returned from this endpoint.